src/content/app/genome-browser/state/browser-sidebar-modal/browserSidebarModalSlice.ts

```diff
@@ -30,7 +30,8 @@ export enum BrowserSidebarModalView {
   BOOKMARKS = 'Previously viewed',
   SHARE = 'Share',
   DOWNLOADS = 'Downloads',
-  NAVIGATE = 'Navigate'
+  NAVIGATE_REGION = 'Navigate region',
+  NAVIGATE_LOCATION = 'Navigate location'
```

Comment:
I can't be 100% certain; but I think the behaviour currently seen on the review deployment is incorrect. Specifically, pressing the button in the toolstrip when the modal is on should probably close the modal regardless of which of the two modals the user had open, whereas on the review deployment, pressing the button when you are on the second modal will take you back to the first one:

https://user-images.githubusercontent.com/6834224/208416013-c8bdd4fc-1dd5-43e5-8255-12c242411351.mov

Reply:
✅